config/example.yaml

```diff
@@ -18,3 +18,4 @@ mod_list:
 output_dir: output
 warnings: ignore
 quiet: false
+float_format: 4
```

Comment:
Its not...yet but I expected it might be in the future. Happy to remove until such time, or pull it out after discussions on formatting have decided how to handle.